### PR TITLE
3.X.X: Mostly refinements (develop -> master)

### PR DESCRIPTION
**Pull request by ethan-nelson**
We're working on a release with more of our Hacktoberfest PRs (as well as some that were collecting some dust). Most of these relate to feature or visual refinements:

* Reduce the high amount of whitespace at page tops (#1261)
* Add translation syntax to some strings that were missing it (#1235)
* Update some language translations (#1254)
* Add page names to the `<title>` so tabs and history lists can be navigated (#1252)
* Allow tasks on square grid that has been clipped to be split (*only for projects created hereafter*) (#1216)
* Fix query on user profiles that returns the projects mapped and validated (#1192)
* Allow for turnkey customization of auto unlock timer (#1274)
* Add additional broad-brush statistics (#1172)

Please provide feedback! All of these changes are on the TM at https://tasks-stage.hotosm.org. We are looking to merge around Dec. 14.

### CODE REVIEW

**Comment by dakotabenjamin** [comment]
Given that there is a db migration needed, I recommend we do a minor version update (v3.1.0)

The new homepage stats haven't been added to the homepage UI yet. Is there a plan to add those? Or are they going to be displayed somewhere else?

Lastly, can someone with an project manager account create a new task on the staging server or give me permissions for that?

**Comment by smit1678** [comment]
> The new homepage stats haven't been added to the homepage UI yet. Is there a plan to add those? Or are they going to be displayed somewhere else?

They won't be added to front page to be viewable. Just API accessible for now. 

> Lastly, can someone with an project manager account create a new task on the staging server or give me permissions for that?

Done.

**Comment by ethan-nelson** [comment]
Thanks @dakotabenjamin for the review. Regarding the user profile, it looks like it does work on stage but just takes a while--I'm guessing that is because it is a small DB instance. https://tasks-stage.hotosm.org/user/FTA loaded for me after about ~30 seconds. We probably need to revisit the task time calculation again at some point, or perhaps load that _after_ the table since it is a bit more complex.

**Comment by ethan-nelson** [comment]
I'll do a run through tonight on staging as well.

**Comment by dakotabenjamin** [comment]
A lesson in patience :grin:

File: client/app/about/about.html [hunk 1]
```diff
@@ -12,7 +12,9 @@ <h1 class="section__title">
...
+                    {{'All of this work is done through'| translate }} <a href="https://www.openstreetmap.org/" target="_top">OpenStreetMap</a>. {{'OpenStreetMap is the community-driven free and editable map of the world, supported by the not-for-profit '| translate }}<a
+                        href="http://wiki.osmfoundation.org/wiki/Main_Page" target="_top">{{'OpenStreetMap Foundation'| translate }}</a>. {{'Read more on the'| translate }} <a href="https://wiki.openstreetmap.org/wiki/Main_Page">{{'OSM Wiki'| translate }}</a>{{' or join the discussion with your '| translate }}<a
+                        href="http://wiki.openstreetmap.org/wiki/Mailing_lists#Country-Specific_Lists">{{'local OSM community'| translate }}</a>.
```
Comment (smit1678) [inline client/app/about/about.html]: Just noticed this when testing on tasks-stage, missing some spaces between links and text: 
about - hot tasking manager 2018-12-12 17-56-09

File: client/app/about/faq.html [hunk 2]
```diff
@@ -6,51 +6,57 @@ <h1 class="section__title">
...
             </div>
         </header>
         <div class="section-container">
             <div class="section__body section__body--secondary">
                 <div class="inner">
-                    <h3>What is Tasking Manager 3 (TM3)?</h3>
+                    <h3>{{'What is Tasking Manager 3 (TM3)?'| translate }}</h3>
                     <p>
-                        A new and improved version of the Tasking Manager was developed by HOT with support by USAID and DFAT during 2017. More information about what’s new in TM3 can be found here: <a href="http://tm3.hotosm.org/what-is-new">http://tm3.hotosm.org/what-is-new</a>.
+                        {{'A new and improved version of the Tasking Manager was developed by HOT with support by USAID and DFAT during 2017. More information about what’s new in TM3 can be found here: '| translate }}<a
```
Comment (smit1678) [inline client/app/about/faq.html]:
```suggestion
                        {{'A new and improved version of the Tasking Manager was developed by HOT with support by USAID and DFAT during 2017. More information about what’s new in TM3 can be found here:  '| translate }}<a
```

File: client/app/about/about.html [hunk 3]
```diff
@@ -12,7 +12,9 @@ <h1 class="section__title">
...
                 </p>
                 <p>
-                    All of this work is done through <a href="https://www.openstreetmap.org/" target="_top">OpenStreetMap</a>. OpenStreetMap is the community-driven free and editable map of the world, supported by the not-for-profit <a href="http://wiki.osmfoundation.org/wiki/Main_Page" target="_top">OpenStreetMap Foundation</a>. Read more on the <a href="https://wiki.openstreetmap.org/wiki/Main_Page">OSM Wiki</a> or join the discussion with your <a href="http://wiki.openstreetmap.org/wiki/Mailing_lists#Country-Specific_Lists">local OSM community</a>.
+                    {{'All of this work is done through'| translate }} <a href="https://www.openstreetmap.org/" target="_top">OpenStreetMap</a>. {{'OpenStreetMap is the community-driven free and editable map of the world, supported by the not-for-profit '| translate }}<a
+                        href="http://wiki.osmfoundation.org/wiki/Main_Page" target="_top">{{'OpenStreetMap Foundation'| translate }}</a>. {{'Read more on the'| translate }} <a href="https://wiki.openstreetmap.org/wiki/Main_Page">{{'OSM Wiki'| translate }}</a>{{' or join the discussion with your '| translate }}<a
```
Comment (smit1678) [inline client/app/about/about.html]:
```suggestion
                        href="http://wiki.osmfoundation.org/wiki/Main_Page" target="_top">{{'OpenStreetMap Foundation'| translate }}</a>. {{'Read more on the'| translate }} <a href="https://wiki.openstreetmap.org/wiki/Main_Page"> {{'OSM Wiki'| translate }}</a>{{' or join the discussion with your '| translate }}<a
```

File: client/app/about/faq.html [hunk 4]
```diff
@@ -6,51 +6,57 @@ <h1 class="section__title">
...
+                            href="http://tm3.hotosm.org/what-is-new">http://tm3.hotosm.org/what-is-new</a>.
```
Comment (smit1678) [inline client/app/about/faq.html]:
```suggestion
                            href="http://tm3.hotosm.org/what-is-new"> http://tm3.hotosm.org/what-is-new</a>.
```

File: client/app/about/about.html [hunk 5]
```diff
@@ -12,7 +12,9 @@ <h1 class="section__title">
...
                 </p>
                 <p>
-                    All of this work is done through <a href="https://www.openstreetmap.org/" target="_top">OpenStreetMap</a>. OpenStreetMap is the community-driven free and editable map of the world, supported by the not-for-profit <a href="http://wiki.osmfoundation.org/wiki/Main_Page" target="_top">OpenStreetMap Foundation</a>. Read more on the <a href="https://wiki.openstreetmap.org/wiki/Main_Page">OSM Wiki</a> or join the discussion with your <a href="http://wiki.openstreetmap.org/wiki/Mailing_lists#Country-Specific_Lists">local OSM community</a>.
+                    {{'All of this work is done through'| translate }} <a href="https://www.openstreetmap.org/" target="_top">OpenStreetMap</a>. {{'OpenStreetMap is the community-driven free and editable map of the world, supported by the not-for-profit '| translate }}<a
```
Comment (smit1678) [inline client/app/about/about.html]:
```suggestion
                    {{'All of this work is done through'| translate }} <a href="https://www.openstreetmap.org/" target="_top"> OpenStreetMap</a>. {{'OpenStreetMap is the community-driven free and editable map of the world, supported by the not-for-profit '| translate }}<a
```

File: client/app/about/about.html (hunk 3 above)
Comment (smit1678) [inline client/app/about/about.html]:
```suggestion
                        href="http://wiki.osmfoundation.org/wiki/Main_Page" target="_top"> {{'OpenStreetMap Foundation'| translate }}</a>. {{'Read more on the'| translate }} <a href="https://wiki.openstreetmap.org/wiki/Main_Page">{{'OSM Wiki'| translate }}</a>{{' or join the discussion with your '| translate }}<a
```

Comment (dakotabenjamin) [reply client/app/about/about.html]: @smit1678 unfortunately due to our branch protection policy this method of applying changes will not work. I will pull these together into a PR foryou or @ethan-nelson to review and merge into `develop` (automatically putting that into this PR)

**Comment by ethan-nelson** [comment]
Thanks @smit1678 for that catch. I added some comments in slack that I hope we can find a better way in general for translating strings that include a link other than putting the html in the string.

I retested these changes and they are working fine for me (since I can't approve my own PR)

**Comment by dakotabenjamin** [comment]
I'll plan to merge tomorrow night (2018-12-15 0000GMT) unless something is brought up before then.

**Comment by smit1678** [comment]
Another flag from testing, seeing #1282 pop up when creating new projects and then testing the split functionality. Posted a screen recording of this in Slack. Not sure if this popped up when we merged a recent PR or whether it wasn't raised when merging in #1216.

**Comment by dakotabenjamin** [comment]
How did you reproduce the bug? Does it always happen for you?

It would seem that this bug is already present in `master`, but if something has changed then we should look into it.

**Comment by smit1678** [comment]
@dakotabenjamin I was testing the non-square tasks being split. This isn't available in master. But I did just test square tasks in master and it looks like that issue is there now. 

To reproduce: 
1. Create a new project with a square grid
2. Clip grid to input polygon to get non-square tasks on edges
3. Start mapping one of the non-square tasks and then select Split Task

**Comment by dakotabenjamin** [comment]
Hm, I can't get that to reproduce on projects I've created, but I can see the issue on your project: https://tasks-stage.hotosm.org/project/5538

I even tried making a project with the same boundaries as your lisbon one but couldn't get it to malfunction.

What should we do in this case?

**Comment by smit1678** [comment]
> Hm, I can't get that to reproduce on projects I've created

New projects or ones that you created a while back? 

@ramyaragupathy had reproduced this, did you do anything specific?

**Comment by dakotabenjamin** [comment]
https://tasks-stage.hotosm.org/project/5544

**Pull request merged**